docs/index.md

```diff
@@ -43,6 +57,27 @@ core_lib:
         protocol: sqlite
   ...
 ```
+#### `your_core_lib.yaml` Explained:
+<b>your_core_lib.yaml is the setting for your entire Core-Lib library. The above example will show how to configure core-lib to connect to a database using [SQLAlchemy](https://docs.sqlalchemy.org/en/20/):</b>
+- <b>log_queries:</b>False, meaning that SQL queries won't be logged. True, [echo flag](https://docs.sqlalchemy.org/en/20/core/engines.html#more-on-the-echo-flag) is set to true.
+- <b>create_db:</b> Create all tables stored in this project-defined entities metadata.[1](https://docs.sqlalchemy.org/en/20/core/metadata.html#sqlalchemy.schema.MetaData.create_all)
+- <b>session:</b> Configuration options related to SQLAlchemy sessions, which represent a "workspace" for interacting with the database.
```

Comment:
session will group all sqlalchemy session setup definitions. / advanced setting.
to define this please read sqlalchmey before that